app/lint.xml

```diff
@@ -81,15 +81,15 @@
   <!-- Ignore Monochrome icon introduced in Android 12 -->
   <issue id="MonochromeLauncherIcon" severity="informational" />
 
+  <!-- Ignore using Material with Material3 Library -->
+  <issue id="UsingMaterialAndMaterial3Libraries" severity="ignore" />
+
   <!--
   Lint complains that the tag is not supported for generated images < api 24.
 
   This was checked on a Lollipop (21) emulator and the images seem to render fine.
   -->
   <issue id="VectorRaster">
-    <ignore path="src/main/res/drawable/ic_settings_account_24dp.xml" />
-    <ignore path="src/main/res/drawable/ic_settings_phone_24dp.xml" />
-    <ignore path="src/main/res/drawable/ic_settings_24dp.xml" />
     <ignore path="src/main/res/drawable/ic_onboarding_intro_1.xml" />
```

Comment:
Can we also remove `<ignore path="src/main/res/drawable/ic_language_24dp.xml" />` from the file?